build/ctags.go

```diff
@@ -49,7 +49,7 @@ func ctagsAddSymbolsParserMap(todo []*zoekt.Document, languageMap ctags.Language
 	var tagsToSections tagsToSections
 
 	for _, doc := range todo {
-		if doc.Symbols != nil {
+		if doc.Content == nil || doc.Symbols != nil {
```

Comment:
nit: rather check that `len(doc.Content) == 0`.  Means the same thing but you get the benefit of avoiding ctags for empty content.